dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/boogie/SvcompProcedures.java

```diff
@@ -21,12 +21,8 @@
 
 public class SvcompProcedures {
 
-	static List<String> FENCES = Arrays.asList("After_atomic", "Before_atomic", "Isync" ,"Lwsync" ," Mb", "Mfence", 
-										"Rcu_lock" , "Rcu_unlock", "Rmb", "Sync", "Sync_rcu","Wmb", "Ish");
-
```

Comment:
Why are these changes in this PR? Are there even SVComp benchmarks that use fences?

Reply:
Same answer as the other PR ... related to a student project (there are no fences in SVCOMP).